editor/src/components/inspector/nine-block-controls.tsx

```diff
@@ -174,7 +174,7 @@ const NineBlockControlCell = React.memo<NineBlockControlCellProps>((props) => {
 
   return (
     <div
-      onClick={onClick}
+      onMouseDown={onClick}
```

Comment:
nit: maybe worth renaming the `onClick` prop as well? either `onMouseDown` or a more specific `onOpen` or something?

Reply:
Great idea! I ended up using `onSelect`